src/options/view.rs

```diff
@@ -265,6 +266,14 @@ impl TimeFormat {
     }
 }
 
+impl UserFormat {
+    fn deduce(matches: &MatchedFlags<'_>) -> Result<Self, OptionsError> {
+        let flag = matches.has(&flags::NUM_UGID)?;
+        Ok(if flag { Self::Numeric } else { Self::Name })
+    }
+
+}
```

Comment:
Just a style nitpicking:

```suggestion
}


impl UserFormat {
    fn deduce(matches: &MatchedFlags<'_>) -> Result<Self, OptionsError> {
        let flag = matches.has(&flags::NUM_UGID)?;
        Ok(if flag { Self::Numeric } else { Self::Name })
    }
}
```